### DISCUSSION TITLE
https://imagekit.io/docs/arithmetic-expressions-in-transformations

### DISCUSSION THREAD

**Discussion by giscus** [imagekitio/discussions#36] [Docs]
# https://imagekit.io/docs/arithmetic-expressions-in-transformations

Explore our guides and examples to integrate ImageKit

https://imagekit.io/docs/_next/static/chunks/pages/_app-9c459202b0c188b1.js

**Comment by MSW-VA** [comment]
I'm a new user...These options should be valuable.

Is there a way to RETURN one or more parameter values for the original (pre-transformation) image asset using syntax analogous to that used in generating links to transformed image data via URL-based transformation parameters?  Might such a RETURN value/list/array be accessible via 3rd-party web data access functions such as MS Excel's WEBSERVICE function?  If not, consider this a feature request.

Thx
